pages/common/hg-push.md

```diff
@@ -0,0 +1,27 @@
+# hg push
+
+> Push changes from the local repository to a specified destination.
+
+- Push changes to the "default" remote path:
+
+`hg push`
+
+- Push changes to a specified remote repository:
+
+`hg push {{path/to/destination_repository}}`
+
+- Allow pushing a new branch (disabled by default):
```

Comment:
Don't we need to specify the branch name here ?

Reply:
Not necessarily, it can be used along with `--branch {{branch}}` though.

`--new-branch` will just allow pushing any new branches.

Reply:
Ok, IIUC this will create the branch if it does not exist in the server previously, right ? I gont a bit confused earlier because of "pushing a branch". Perhaps "Push a new branch if it does not exist" is more clear ?

Reply:
Yeh, that makes more sense. 👍